doorway-ui-components/src/global/mixins.scss

```diff
@@ -143,9 +143,9 @@
 
 @mixin outlined-appearances() {
   background-color: var(--bloom-color-white);
-  color: var(--bloom-color-primary);
+  color: var(--bloom-color-primary-dark);
```

Comment:
(q, nonblocking) FYI I don't see this mixin used anywhere in this PR so I assume it just fixes something that looks bad somewhere else?

Reply:
Button uses this mixin, and changing this affects the button that says "Clear all filters", specifically the border color and the text color.